build/python/backend/Dockerfile

```diff
@@ -134,6 +134,7 @@ RUN cd /strelka/ && \
     git \
     libtool \
     make \
+    unrar \
```

Comment:
FYSA: needed to remove `unrar` after using it as it has a vuln (added after your reviewed @cameron-dunn-sublime).

Reply:
Does that interrupt https://github.com/sublime-security/strelka/blob/7229988c119e5e3c9b496f64d9c73c174515f589/src/python/strelka/scanners/scan_rar.py#L17?

Reply:
good catch! let's see 😟

Reply:
It's not in their report, and we're on the latest available in our package manager. If they come back and complain about it I'll have to compile from source...